### PR TITLE
test with pekko rc2

### PR DESCRIPTION
**Pull request by pjfanning**
useful to try the pekko rc2 jars in automated builds

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -29,6 +29,7 @@ inThisBuild(Def.settings(
   scmInfo := Some(
     ScmInfo(url("https://github.com/apache/incubator-pekko-http"), "git@github.com:apache/incubator-pekko-http.git")),
   description := "Apache Pekko Http: Modern, fast, asynchronous, streaming-first HTTP server and client.",
+  resolvers += "Apache Pekko Staging".at("https://repository.apache.org/content/groups/staging"),
```
[the remark this answers is not in the document]
Comment (pjfanning) [reply build.sbt:33]: added a TODO

File: project/PekkoDependency.scala [hunk 2]
```diff
@@ -53,7 +53,7 @@ object PekkoDependency {
   }
 
   // Default version updated only when needed, https://pekko.apache.org/docs/pekko/current/project/downstream-upgrade-strategy.html
-  val minimumExpectedPekkoVersion = "0.0.0+26720-01379c41-SNAPSHOT"
+  val minimumExpectedPekkoVersion = "1.0.0-rc1"
```
Comment (mdedetrich) [inline project/PekkoDependency.scala:56]: Is it possible to capatlize the RC here, i.e. "1.0.0-RC1". Even if it doesn't make a difference, I have never seen a lowercase convention when it comes to OS Scala RC's.

Comment (pjfanning) [reply project/PekkoDependency.scala:56]: the files are named with rc1 - so this is more correct

please see https://repository.apache.org/content/groups/staging/org/apache/pekko/pekko-actor_3/1.0.0-rc1/

Comment (mdedetrich) [reply project/PekkoDependency.scala:56]: https://lists.apache.org/thread/4hkt05tsqc82jr1oqsw39or0pwxk2ym4

**Review by mdedetrich — APPROVED** [review]
lgtm

**Pull request merged**